Doc/includes/email-alternative.py

```diff
@@ -15,7 +15,7 @@
 msg.set_content("""\
 Salut!
 
-Cela ressemble à un excellent recipie[1] déjeuner.
+Cela resemble à un excellent recipie[1] déjeuner.
```

Comment:
This is a correctly spelled French word in a French sentence.

https://en.wiktionary.org/wiki/ressemble

Also below, please don't change them.